cmd/prow-job-dispatcher/main.go

```diff
@@ -48,7 +48,7 @@ func gatherOptions() options {
 	fs.StringVar(&o.prometheusURL, "prometheus-url", "https://prometheus-prow-monitoring.apps.ci.l2s4.p1.openshiftapps.com", "The prometheus URL")
 	fs.StringVar(&o.prometheusUsername, "prometheus-username", "", "The Prometheus username.")
 	fs.StringVar(&o.prometheusPasswordPath, "prometheus-password-path", "", "The path to a file containing the Prometheus password")
-	fs.StringVar(&o.jobVolumesPath, "job-volumes-path", filepath.Join(os.TempDir(), "job-volumes.yaml"), "The path to a file containing the job volumes")
+	fs.IntVar(&o.prometheusDaysBefore, "prometheus-days-before", 0, "Number [0,15] of days before yesterday. 00-00-00 of that day will be used as time to query Prometheus. E.g., 0 means 00-00-00 yesterday.")
```

Comment:
days of *what* before yesterday? "Use last X days of metrics from Prometheus".

Reply:
See if it is better now.